src/Admin/Controllers/OrganizationsController.cs

```diff
@@ -137,7 +149,12 @@ public async Task<IActionResult> View(Guid id)
         }
         var users = await _organizationUserRepository.GetManyDetailsByOrganizationAsync(id);
         var billingSyncConnection = _globalSettings.EnableCloudCommunication ? await _organizationConnectionRepository.GetByOrganizationIdTypeAsync(id, OrganizationConnectionType.CloudBillingSync) : null;
-        return View(new OrganizationViewModel(organization, provider, billingSyncConnection, users, ciphers, collections, groups, policies));
+        var secrets = organization.UseSecretsManager ? await _secretRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<Secret>();
+        var projects = organization.UseSecretsManager ? await _projectRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<Project>();
+        var serviceAccounts = organization.UseSecretsManager ? await _serviceAccountRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<ServiceAccount>();
```

Comment:
We also need to show the occupied SM seats. It looks like this is missing from the Figma, so you may need to check back with Sukhleen.

Reply:
I have added the secrets manager seats count while awaits Sukhleen to Update the UI